other/disallow-all-secrets/disallow-all-secrets.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: kyverno.io/v1
-kind: Policy
+kind: ClusterPolicy
```

Comment:
Have not updated the Artifact Hub metadata file.

Reply:
Will update.